spec/features/cookie_banner_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 describe 'cookie banner', type: :feature do
+  include StubbedRequests::Courses
```

Comment:
Probably this comes down to whether the team prefers implicit  vs explicit declaration but one option would be to include the helper via the `spec_helper` so that you do not need to add a list of `includes` in lots of specs. 

```
# spec_helper.rb
RSpec.configure do |c|
  c.include StubbedRequests::Courses
  c.include StubbedRequests::Providers
  # etc
end
```

https://relishapp.com/rspec/rspec-core/v/3-8/docs/helper-methods/define-helper-methods-in-a-module

Reply:
Yea this was one thought I had when reviewing too. Normally I'd be on the side of making it really explicit, but they're pretty consistently used in the feature specs so it might be worth just including in the spec helper.

I think either fine though tbh.

Reply:
I quite like the explicit inclusions at the top of the file - they're informative in that they give a quick overview of which requests are being stubbed and the overall degree of stubbing in a given spec file. Perhaps one to refactor away if it becomes too laborious?